environment.yml

```
@@ -6,12 +6,12 @@ dependencies:
   - astropy>=5,<6
   - black
   - filelock>=3,<4
-  - frictionless>=4.29,<5
+  - frictionless>=5.10.1,<6
   - isort
   - jupytext>=1.13,<2
   - matplotlib-base>=3.5.0,<4
-  - myst-nb>=0.13.0,<0.14.0
-  - myst-parser>=0.15,<0.17
+  - myst-nb>=0.15.0,<0.18
+  - myst-parser>=0.15,<0.19
```

```suggestion
  - myst-nb>=0.17.0,<0.18
  - myst-parser>=0.18,<0.19
```

Since these are just for building the docs, I would pin them rather strictly.